README.md

```diff
@@ -1,2 +1,15 @@
 # ExpenseTrackerGrupo5
 Take control of your finances.
+
+
+### Levantar contenedores 
+
+```bash
+docker-compose up -d
+```
+
+### Create and apply migrations
```

Comment:
Do we still need this step?

Reply:
Sure you have the reason this is not necessary to do